File: cpp/sobrasLucroEmpresa.cpp

#include <iostream>
#include <locale.h>
#include <string>

using namespace std;

int main(){
	setlocale(LC_ALL, "");
	
	int lucro, funcionarios, sobras, balinhas;
	double preco;

	cin >> lucro >> funcionarios >> preco;
	sobras = lucro % funcionarios;
	balinhas = sobras / preco;

	cout << balinhas;
	return 0;
}	

File: java/sobrasLucroEmpresa.java
/*
 * Laboratório de Programação 2 - Lab 1
 *
 * @author Franklin Regis de Oliveira - 119210030
 */

import java.util.Scanner;

public class sobrasLucroEmpresa{
	public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
		int lucro = sc.nextInt();
		int func = sc.nextInt();
		int sobras = lucro % func;
		double preco = sc.nextDouble();
		int balinhas = (int) (sobras / preco);
		System.out.println(balinhas);

	}
}
